src/geometry/ArborX_KDOP.hpp

```diff
@@ -227,9 +228,6 @@ struct expand<KDOPTag, PointTag, KDOP, Point>
 {
   KOKKOS_FUNCTION static void apply(KDOP &kdop, Point const &point)
   {
-    using Details::KokkosExt::max;
-    using Details::KokkosExt::min;
```

Comment:
Please be consistent within the same file.
Either do using-declaration + unqualified calls or everywhere qualified calls.
Don't mix unless there is a good reason to.